config/fcos/v1_5_exp/translate.go

```diff
@@ -16,6 +16,7 @@ package v1_5_exp
 
 import (
 	"fmt"
+	"log"
```

Comment:
probably a better way to do this without importing log will take a look for a better solution